apps/web/components/getting-started/steps-views/SetupAvailability.tsx

```diff
@@ -72,8 +111,17 @@ const SetupAvailability = (props: ISetupAvailabilityProps) => {
             });
           }
 
-          mutation.mutate({ completedOnboarding: true });
+          await Promise.all(
+            DEFAULT_EVENT_TYPES.map(async (event) => {
+              return createEventType.mutate(event);
+            })
+          );
+
+          await utils.viewer.me.refetch();
+
+          await profileMutation.mutate({ completedOnboarding: true });
         } catch (error) {
+          setIsFinishClicked(false);
```

Comment:
use "finally" block to set false value.

Reply:
As discussed could be error prone (in case of slow connection), so for now I'll leave it as is and refactor it later.